doc/source/myrocks/column-family.rst

```diff
@@ -0,0 +1,108 @@
+.. _ps.myrocks.column-family:
+
+================================================================================
+|myrocks| Column Families
+================================================================================
+
+
+|MyRocks| stores all data in a single database instance as a collection of
+key-value pairs. This is a flat data structure that requires that keys be unique
+throughout the database. |MyRocks| incorporates table IDs and index IDs into
+the keys.
```

Comment:
This is misleading.  A reader can easily conflate a MyRocks 'database' with a MySQL 'database'. A more correct wording is something like "MyRocks stores all of it data in a single server instance within a data structure known as a Log Structured Merge (or LSM) tree as a collection of key-value pairs.  This is a flat data structure that requires that keys be unique throughout the data structure." or something like that.  Otherwise a casual reader may take away that each individual MySQL 'database' is an individual MyRocks 'thing'.